release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -3023,6 +3023,60 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+[id="ocp-4-18-22-known-issues_{context}"]
+==== Known issues
+
+* Stale Source Network Address Translations (SNATs) or routing policies might occur in the following circumstances:
+** You are upgrading from 4.17 to 4.18 during an update of the OVN-Kubernetes image.
+** During the upgrade, if a pod on another system that was selected by an egress IP was deleted when the `ovnkube-node` pod was not running. (link:https://issues.redhat.com/browse/OCPBUGS-59531[OCPBUGS-59531])
+
+[id="ocp-4-18-22-bug-fixes_{context}"]
+==== Bug fixes
+
+* Before this update, destroying a cluster in the unsupported region `mx-central-1` caused the destroyer to fail to find a partition and not exit. As a consequence, you could not destroy an {product-title} cluster in the `mx-central-1`region due to constant error reporting. With this release, the destroyer does not report errors for the unsupported region `mx-central-1`, which enables the successful destruction of a cluster. (link:https://issues.redhat.com/browse/OCPBUGS-56177[OCPBUGS-56177])
+
+* Before this update, combined specification and status updates lists triggered unnecessary firmware upgrades, which caused system downtime. With this release, a firmware upgrade optimization skips unnecessary firmware upgrades. (link:https://issues.redhat.com/browse/OCPBUGS-56766[OCPBUGS-56766])
+
+* Before this update, the `console-telemetry` plugin received a `Forbidden` error due to using the wrong API endpoint for tracking usage. As a consequence, the `Forbidden error in console-telemetry-plugin usage tracking` error occurred. With this release, the `console-telemetry` plugin posts usage data to `/api/metrics/usage` instead of `/metrics/usage`. As a result, the `console-telemetry`plugin does not receive a `Forbidden` error, which ensures accurate usage tracking. (link:https://issues.redhat.com/browse/OCPBUGS-58364[OCPBUGS-58364])
+
+* Before this update, the installer failed when {aws-first} credentials were not found and the survey was attempting to list all {aws-short} regions preventing users from creating the install-config. With this release, the installer no longer fails when {aws-short} credentials are not set, allow users to input them during the survey. (link:https://issues.redhat.com/browse/OCPBUGS-59155[OCPBUGS-59155])
+
+* Before this update, when a `HostedCluster` was configured with a proxy URL such as http://user:pass@host, the authentication header was not getting forwarded by the konnectivity proxy to the user proxy, failing authentication. With this release, the proper authentication header is sent when a user and password is specified in the proxy URL. (link:https://issues.redhat.com/browse/OCPBUGS-59503[OCPBUGS-59503])
```

Comment:
Items in backticks need a noun to follow. Can this just be "hosted cluster" to simplify that? 

You also need to escape the URL so it's not clickable text.